packages/core/template/template.js

```diff
@@ -5,46 +5,50 @@ const fsUtil = require('../src/utils/fsUtil');
 
 const requiredFiles = ['index.md', 'site.json', '_markbind/'];
 
-function Template(rootPath, templatePath) {
-  this.rootPath = rootPath;
-  this.template = templatePath;
-}
+class Template {
+  constructor(rootPath, templatePath) {
+    this.root = rootPath;
+    this.template = templatePath;
+  }
 
-function validateTemplateFromPath(templatePath) {
-  for (let i = 0; i < requiredFiles.length; i += 1) {
-    const requiredFile = requiredFiles[i];
-    const requiredFilePath = path.join(templatePath, requiredFile);
+  static validateTemplateFromPath(templatePath) {
```

Comment:
Should this be static?

Reply:
Ah right, I'll change them back (they were changed to `static` while I was trying to debug some cases where the functions weren't being referenced properly). Same for the other method.

Thanks for pointing it out!

Reply:
Looks like there was a reason why I didn't change them back... Having them not be `static` methods will cause the Eslint to complain, as detailed [in the Eslint docs](https://eslint.org/docs/latest/rules/class-methods-use-this), and shown in the screenshot below:

![Screenshot from 2023-02-09 23-22-54](https://user-images.githubusercontent.com/25716529/217855466-5851b061-ca1d-4a0b-8cc1-3915cb517507.png)

The reasoning in the Eslint docs seem to be that as long as a method does not make use of the `this` keyword to reference the object, there is no reason why it should be an instance method.

Reply:
And my suggestion follows: how about making the parameters instance variables?

Reply:
Updated according to the suggestion in [ffa06c4](https://github.com/MarkBind/markbind/pull/2148/commits/ffa06c4634c60a65c7aebc0d9adf46b6152df31c)!